src/main/java/frc/robot/RobotMap.java

```diff
@@ -24,7 +24,10 @@
     public static final int SHOOTER_MOTOR_2 = 42;
 
     //TURRET
-    public static final int TURRET_MOTOR = -1; //Set this to real value ASAP
+    public static final int TURRET_MOTOR = LEFT_CLIMBER_MOTOR; //Set this to real value ASAP
```

Comment:
Is this a real value now? Can we remove comment?

Reply:
Not a real value yet as turret is still using the climber motor + wood block for testing